cpu/samd5x/periph/can.c

```diff
@@ -214,6 +215,14 @@ static int _set_mode(Can *can, can_mode_t can_mode)
 #endif
             _exit_init_mode(can);
             break;
+        case MODE_MONITOR:
```

Comment:
monitor mode should be enterd on initialization (only left afterwards)  
a device that is supposed to just monitor needs to just do that from the start.